recommendation/views.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.sites.shortcuts import get_current_site
+from django.core.mail import EmailMessage
+from django.shortcuts import render, redirect
+from django.template.loader import render_to_string
+from django.http import HttpResponse
+from .forms import RecommendationForm
+from register.models import Student
+from OneApply.constants import UserType
+
+
+def new_recommendation(request):
+    if request.method == "POST":
+        user_type = request.session.get("user_type", None)
+        username = request.session.get("username", None)
```

Comment:
Adding test cases covering the unauth access to recommendations tab (something similar to what has been done for all_schools) will provide for a more robust build. Having said that, please add a TODO instead if the time doesn't permit adding them rn.

Reply:
TODO added